Remotion/Web/Development.WebTesting/Accessibility/Implementation/AccessibilityResultMapper.cs

```diff
@@ -46,7 +46,14 @@ public class AccessibilityResultMapper : IAccessibilityResultMapper
             "cat.structure",
             "experimental",
             "review-item",
-            "best-practice"
+            "best-practice",
+            "ACT"
+        };
+
+    private readonly HashSet<string> _wellKnownSuccessCriteraPrefixesToIgnore = new HashSet<string>
+        {
+            "TT",
+            "EN",
```

Comment:
Replace with array of regex that matches for "EN-..." and "TT\d{1-2}..."
Reasoning: a bit more precision in what we are doing.